src/main.rs

```diff
@@ -0,0 +1,57 @@
+use rand::Rng;
+use std::fmt;
+
+fn main() {
+    let mut bag = TileBag::new_default();
+    for i in 0..1000 {
+        let tile = bag.draw_tile();
+        println!("{}", tile)
+    }
+}
+
+struct TileBag {
+    bag: Vec<char>,
+    rng: rand::rngs::ThreadRng,
+}
+
+impl TileBag {
+    fn new(letter_distribution: [u8; 26]) -> Self {
+        let mut bag: Vec<char> = vec![];
```

Comment:
```suggestion
        let mut bag = vec![];
```

It's rare that you have to specify a type in Rust, in almost all cases it can be inferred from usage (i.e. pushing a char to the vec types it)

Reply:
(though as reflected below I'd remove this declaration anyway)